frontend/online-shop/src/services/orders.service.ts

```diff
@@ -0,0 +1,8 @@
+import axios from "axios"
+import { CREATE_ORDER_ENDPOINT } from "../helpers/strings"
+
+
+export const createOrder =async ({customer, products}: any)=>{
```

Comment:
Extract the props in a separate local interface. Also don't use any types unless there is no other way